pkg/testUtils/testUtils.go

```diff
@@ -0,0 +1,26 @@
+package testutils
+
+import (
+	"path/filepath"
+
+	"github.com/massalabs/station-massa-wallet/pkg/wallet"
+	"github.com/massalabs/station/pkg/logger"
+)
+
+const (
+	LogFileNameTest = "station-massa-wallet-test.log"
+)
+
+func LoggerTest() error {
+	logPath, err := wallet.Path()
```

Comment:
I would change that to write the logs in the repository directly
Would be way easier to check the logs of tests

Reply:
ho yes!